plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller.rs

```diff
@@ -42,12 +42,18 @@ enum LogitechBulkcontrollerCmd {
     Nack = 0xFF03,
 }
 
-#[derive(New)]
+#[derive(New, ToString, Getters)]
 struct LogitechBulkcontrollerSendSyncReq {
     cmd: LogitechBulkcontrollerCmd,
     payload_length: u32le,
     sequence_id: u32le,
 }
+#[derive(Parse)]
+struct LogitechBulkcontrollerSendSyncRes {
+    cmd: LogitechBulkcontrollerCmd,
+    payload_length: u32le,
+    sequence_id: u32le,
+}
 
```

Comment:
Not familiar with .rs file syntax. Is "_" intended here?
    _payload_length: u32le,

Reply:
Yup, that just means "reserved" -- i.e. we don't care.